app/assets/stylesheets/moderators.scss

```diff
@@ -155,7 +155,39 @@
 }
 
 .mod-container {
-  padding: 30px 0px;
+  padding: 16px 0px;
+
+  header.top-header {
+    padding: 0px 16px;
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    h1 {
+      font-size: 18px;
```

Comment:
We also have font-size variables with `crayons`. Instead of `18px`, we can say: `$fs-l`